t/18-backend-qemu.t

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/perl
+
+use 5.018;
+use strict;
+use warnings;
+
+use Test::More;
+use Test::MockModule;
+use Test::MockObject;
+use Test::Warnings;
```

Comment:
@cfconrad I fixed the warnings with https://github.com/os-autoinst/os-autoinst/pull/1194 and check for no warnings with this line

Reply:
thx for hint!